server/src/main/java/io/unitycatalog/server/persist/dao/ModelVersionInfoDAO.java

```diff
@@ -0,0 +1,101 @@
+package io.unitycatalog.server.persist.dao;
+
+import io.unitycatalog.server.model.ModelVersionInfo;
+import io.unitycatalog.server.model.ModelVersionStatus;
+import io.unitycatalog.server.persist.utils.FileUtils;
+import jakarta.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+// Hibernate annotations
+@Entity
+@Table(
+    name = "uc_model_versions",
+    indexes = {
+      @Index(name = "idx_model_version", columnList = "registered_model_id,version"),
+    })
+// Lombok annotations
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class ModelVersionInfoDAO {
```

Comment:
Unfortunately, because model versions don't have `name` I can't leverage the IndentifiableDao and also means I had to implement a separate page token logic for model versions up in the ModelRepository.